pkgs/tools/package-management/rpm/lint.nix

```diff
@@ -0,0 +1,87 @@
+{ buildPythonPackage
+, checkbashisms # Not in README
+, cpio
+, fetchFromGitHub
+, fetchpatch
+, lib
+, pybeam
+, pyenchant
+, pytestCheckHook
+, pytest-cov
+, pytest-xdist
+, python
+, python-magic
+, pyxdg
+, rpm
+, tomli
+, tomli-w
+, zstandard
+}:
+
+buildPythonPackage rec {
+  pname = "rpmlint";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "rpm-software-management";
+    repo = pname;
+    rev = version;
+    hash = "sha256-NzPGIwbxZ7WyVGxdNTIaYbznvZ4WnFuVEgdwlgmJrx4=";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "drop pytest-flake8";
+      url = "https://github.com/rpm-software-management/rpmlint/pull/1054.patch";
+      hash = "sha256-Pa6wnW+JFYaVqZwZ756dvd7idAU+5r0tZ4IXfmynR/Q=";
+    })
+  ];
+
+  postPatch = ''
+    substituteInPlace test/test_ldd_parser.py --replace 'def test_ldd_parser_failure():' '@pytest.mark.skipif(True, reason="Inside sandbox /bin/sh is busybox and does not support bash localized string")
+    def test_ldd_parser_failure():'
+  '';
+
+  preBuild = ''
+    substituteInPlace setup.py --replace "   'rpm'," ""
+  '';
+
+  buildInputs = [
+    pybeam
+    pyenchant
+    python-magic
+    pyxdg
+    rpm
+  ]
+  ++ lib.optioanals (lib.versionOlder python.version "3.11") [ tomli ]
+  ++ [
+    tomli-w
+    zstandard
+  ];
+
+  nativeCheckInputs = [
+    checkbashisms
+    cpio
+    pytestCheckHook
+    pytest-cov
+    pytest-xdist
+    rpm
+  ];
+
+  meta = with lib; {
+    # metadata from https://rpmfind.net/linux/RPM/fedora/devel/rawhide/x86_64/r/rpmlint-2.4.0-4.fc38.noarch.html
+    description = "Tool for checking common errors in RPM packages";
+    longDescription = ''
+        rpmlint is a tool for checking common errors in RPM packages. Binary
+        and source packages as well as spec files can be checked.
+      '';
+    homepage = "https://github.com/rpm-software-management/rpmlint";
+    downloadPage = "https://github.com/rpm-software-management/rpmlint/releases";
+    changelog = "https://github.com/rpm-software-management/rpmlint/releases/tag/${version}";
+    license = licenses.gpl2Only;
+    maintainers = [];
+    mainProgram = "rpmlint"; # Most tutorials only explain this tools but it is used one time
+    platforms = platforms.unix;
+    sourceProvenance = with sourceTypes; [ fromSource ];
```

Comment:
Weird it is not `soureProvenances`:

```

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/pkgs/stdenv/generic/check-meta.nix:91:17:

           90|
           91|   isNonSource = sourceTypes: lib.lists.any (t: !t.isSource) sourceTypes;
             |                 ^
           92|
```

https://gist.github.com/GrahamcOfBorg/1da26d3f25f5e7af18815170d5e04ba2 it was not catched by nixpgs-review